### PR TITLE
fix: Date convert wrong type as number (close #462)

### LINKED ISSUE #462: end.getTime is not a function
## Current Behavior

```js
                    period = datetime.range(
                        viewModel.getStarts(),
                        viewModel.getEnds(),
                        datetime.MILLISECONDS_PER_DAY
                    );
```
image
## Expected Behavior

```
                period = datetime.range(
                    datetime.start(viewModel.getStarts()),
                    datetime.end(viewModel.getEnds()),
                    datetime.MILLISECONDS_PER_DAY
                );
```

### PR DESCRIPTION
**Pull request by dongsik-yoo**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

Fix that the Date instance convert number type.

This patch will also be merged to `feat/dooray` branch.

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/js/common/datetime.js [hunk 1]
```diff
@@ -610,8 +610,11 @@ datetime = {
     convertStartDayToLastDay: function(d) {
         var date = new TZDate(d);
         var isStartOfDay = datetime.isStartOfDay(d);
+        if (isStartOfDay) {
```
Comment (seonim-ryu) [inline src/js/common/datetime.js]: 변수가 한 번만 사용되고 있어서 할당하지 않고 사용해도 될 것 같습니다.
```js
if (datetime.isStartOfDay(d)) {
}
```

File: src/js/common/datetime.js [hunk 2]
```diff
@@ -610,8 +610,11 @@ datetime = {
...
+            date.setDate(date.getDate() - 1);
+        }
 
-        return isStartOfDay ? date.setDate(date.getDate() - 1) : date;
```
Comment (jungeun-cho) [inline src/js/common/datetime.js]: When writing code for the first time
It was my mistake.
It was a problem by returning the number value when calling `setDate`.

**Comment by brosbunbo** [comment]
@jungeun-cho  Could you release this fix? We are having this issue as well.

**Comment by dongsik-yoo** [comment]
@brosbunbo I released [v1.12.9](https://github.com/nhn/tui.calendar/releases/tag/v1.12.9) including this fix.

**Pull request merged**